### PR TITLE
feat(validation): Helper method to validate user input

### PR DESCRIPTION
**Pull request by arungalva**
Added schemas for the parameters. The schema is used by the verifyParams helper method that checks
if user passed all the requiremed parameters and also checks if the parameters sent by the user are
of the expected type

This is a POC for #15. If agreed that this makes sense, we can add the schemas for rest of user input

### CODE REVIEW

File: __test__/helpers.test.js [hunk 1]
```diff
@@ -10,3 +10,34 @@ describe("Test if valid URL(s)", () => {
     expect(isValidURL("http://foo.bar?q=Spaces should be encoded")).toBe(false);
   });
 });
+
+describe("Test if valid Pramas", () => {
```
Comment (yeskunall) [inline __test__/helpers.test.js]: Please make an effort to proof-read what you're pushing online.

Comment (arungalva) [inline __test__/helpers.test.js]: lol sure

File: lib/utils/helpers.js [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
     !!URL.match(
-      /^(https?:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$/
-    )
+      /^(https?:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$/,
+    ),
+  verifyParams: (endpoint_type, user_input) => {
+    // go through each user_input and check if its valid or not
+    const endpoint_valid_params = params_config[endpoint_type];
+
+    // check if user passed all the requiremnts
```
Comment (yeskunall) [inline lib/utils/helpers.js]: More typos.

Comment (arungalva) [inline lib/utils/helpers.js]: need a spell checker for my text editor lol 😂

**Comment by yeskunall** [comment]
Can you please move b34d4fbb01fcdb936ad9f273c0ba07675fa08bc4 to a separate PR? This needs to be addressed immediately, as you can tell from all the HoundCI error. :sweat:

/cc @arungalva

**Comment by yeskunall** [comment]
Closing this PR as per our discussion. Continuing working on your local branch as it is @arungalva and make a new PR. Thanks!

**Pull request closed without merging**